packages/bot-web-ui/src/pages/bot-builder/bot-builder.tsx

```diff
@@ -100,6 +102,17 @@ const BotBuilder = observer(() => {
             },
         });
     };
+    useEffect(() => {
+        setTimeout(() => {
+            if (isSafari()) {
+                const images: NodeListOf<SVGImageElement> = document.querySelectorAll('.blocklyEditableText image');
+                images.forEach((img: SVGImageElement) => {
+                    const newIconUrl = getImageLocation(is_dark_mode_on ? 'arrow-down-white.svg' : 'arrow-down.svg');
+                    img?.setAttributeNS?.('http://www.w3.org/1999/xlink', 'xlink:href', newIconUrl);
+                });
+            }
+        }, 100);
```

Comment:
can we remove this setTimeout ? Also can we do this in the `dbot.js `file ?

Reply:
hey @rupato-deriv changes updated as u suggested,